### PR TITLE
feature/test-sign-in

### PR DESCRIPTION
**Pull request by gecBurton**
## Context

As an Engineer I want a basic test for sign in so that I have confidence that this is working

## Changes proposed in this pull request

* sign-in view now uses a form
* a basic test for the sign-in has been written
* a django specific README has been written

## Things to check

- [ ] I have added any new ENV vars in all deployed environments
- [x] I have tested any code added or changed
- [ ] I have run integration tests

### CODE REVIEW

File: django_app/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# How to run and test the django app
+
+## running and testing the django app locally
+
+1. copy `.env.test` to `django_app/.env`
+2. (re)start postgres and minio `docker-compose up -d db minio` 
+3. open you IDE at `django_app` and run the tests here, or navigate to `cd django_app`
+4. the server can be run locally using `poetry run python manage.py runserver`
+5. test can be run .locally using `poetry run python pytest`
```
Comment (brunns) [inline django_app/README.md:9]: Just `poetry run pytest`?

I'm still getting 2 test failures at the moment.

File: django_app/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# How to run and test the django app
+
+## running and testing the django app locally
+
+1. copy `.env.test` to `django_app/.env`
+2. (re)start postgres and minio `docker-compose up -d db minio` 
+3. open you IDE at `django_app` and run the tests here, or navigate to `cd django_app`
+4. the server can be run locally using `poetry run python manage.py runserver`
```
Comment (brunns) [inline django_app/README.md:8]: I can see the server on http://localhost:8000/ but I'm not seeing static files.

File: django_app/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# How to run and test the django app
+
+## running and testing the django app locally
+
+1. copy `.env.test` to `django_app/.env`
+2. (re)start postgres and minio `docker-compose up -d db minio` 
+3. open you IDE at `django_app` and run the tests here, or navigate to `cd django_app`
```
Comment (brunns) [inline django_app/README.md:7]: Tests run, but weirdly I see one extra failure as opposed to running the tests from the CLI.

File: django_app/README.md (hunk 2 above)
Comment (gecBurton) [reply django_app/README.md:8]: I am going to fix this in another PR @brunns

File: django_app/README.md (hunk 3 above)
Comment (gecBurton) [reply django_app/README.md:7]: agreed, but also another PR

**Review by brunns — APPROVED** [review]
There's a build failure for some reason - might be transient. Otherwise, LGTM.

**Comment by gecBurton** [comment]
> There's a build failure for some reason - might be transient. Otherwise, LGTM.

looks like poetry update required 🤞

**Pull request merged**